lib/widgets/movie_poster.dart

```diff
@@ -21,12 +24,21 @@ class MoviePoster extends StatelessWidget {
           border: Border.all(
             width: borderWidth,
           ),
-          image: DecorationImage(
-            image: AssetImage(
-              pathToPosterImg,
-            ),
-            fit: BoxFit.fill,
-          ),
+        ),
+        child: Image.asset(
+          pathToPosterImg,
+          height: height,
+          width: width,
+          fit: BoxFit.cover,
+          errorBuilder:
+              (BuildContext context, Object exception, StackTrace? stackTrace) {
```

Comment:
trailing comma and format code here please

Reply:
solved